pkg/localhelm/helm3.go

```diff
@@ -169,7 +170,7 @@ func Deploy(st *state.State) error {
 func releasesFilterFn(helmReleases []kubeoneapi.HelmRelease, logger logrus.FieldLogger) func(rel *helmrelease.Release) bool {
 	return func(rel *helmrelease.Release) bool {
 		for _, hr := range helmReleases {
-			if rel.Name == hr.ReleaseName && rel.Namespace == hr.Namespace && rel.Chart.Name() == hr.Chart {
+			if rel.Name == hr.ReleaseName && rel.Namespace == hr.Namespace && rel.Chart.Name() == path.Base(hr.Chart) {
```

Comment:
Unfortunatelly this is not enough. Directory name can be different from chart name, the actual chart name contained in the Chart.yaml file, which we need to YAML unmarshall in case when chart is specified as a directory path.